fdbclient/include/fdbclient/TenantEntryCache.actor.h

```diff
@@ -102,16 +105,104 @@ class TenantEntryCache : public ReferenceCounted<TenantEntryCache<T>>, NonCopyab
 		return tenantList.results;
 	}
 
+	ACTOR static Future<Void> refreshCacheById(int64_t tenantId,
+	                                           TenantEntryCache<T>* cache,
+	                                           TenantEntryCacheRefreshReason reason) {
+		TraceEvent(SevDebug, "TenantEntryCacheIDRefreshStart", cache->id()).detail("Reason", static_cast<int>(reason));
```

Comment:
nit: maybe leave it as `SevInfo` given the line shall be logged only for the miss (one-time)?